rt/features/metrics/pom.xml

```diff
@@ -34,14 +34,33 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-core</artifactId>
+            <optional>true</optional>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Please remove this dependency, not need really (see please the suggested alternative)

Reply:
Fixed.